Changes

```diff
@@ -1,6 +1,9 @@
 MIME-Base32
 ===========
 
+1.302	2016-06-17
```

Comment:
Please don't add release version neither release dates, since I might decide to accumulate changes.